src/lib.rs

```diff
@@ -32,3 +38,9 @@ pub struct ChatInteraction {
     #[cfg_attr(feature = "test", strategy("[A-Za-z0-9., ]*"))]
     pub response: String,
 }
+
+/// Representation of an Atoma node's public key.
```

Comment:
Can you add what type of signature scheme is being used by Atoma nodes?

Reply:
Right now, we support Ed25519 (main), Secp256k1 and Secp256r1